src/CNAH2.CPP

```diff
@@ -1926,11 +1926,11 @@ DBL AH::tsfo(  			// detailed computation of fan-only supply temp
 	DBL _tr1=1, _tr2, /*_wr1,*/ _cr1, _frFanOn=1.;		// sets these using _tr, _wr, _cr, fcc, .
 	{
 		// init leakage. story in puteRa.
-#ifdef AUTOSIZE//7-95
+
 		DBL leakFracOn = Top.tp_pass1A && asFlow  		// fan-on leakage as fraction of supply fan design flow
 						 ?  0				// use 0 during autoSizing pass 1 part A: determining fan design flow
 						 :  (ahROLeak + ahSOLeak)/2.;	// use average of user's supply & return leaks, for balance for now.
-#else
+#if 0
```

Comment:
This is implicitly `#ifndef AUTOSIZE`, and if we're going to keep these around, we should probably include the same comment we have in other places.